### ISSUE TITLE
[Bug] Playback jumps to beginning on connection status change

### ISSUE THREAD

**Issue by SleepyPrince** [eni9889/YT-PP-Issues#56]
iOS Version: 10.2 (jailbroken)
YouTube Version: 12.29.8
YT ++ Tweak Version: 1.4r-66
Crash Report: N/A
Issue:
When playing downloaded videoes, if connection status changes (I.e. Connecting/disconnecting from wifi/cellular network), the playback will jump to the beginning of the video.

**Comment by ThaumicMekanism** [comment]
I have the same issue. I have tested both the jailbroken version and sideloaded version and both have the issue. I have also experienced this issue when joining or disconnecting from a VPN too.

YT Version: 12.28.7 & 12.29.8 (Have both sideloaded and jailbroken apps)
YT ++ Tweak Version: 1.4r-57 & 1.4r-69 respectively
Crash Report: NA
Issue:
Same as above. I have had this issue ever since I started using youtube ++ and it happens both on the phone and ipad. (Im using an iphone 7+ and ipad Mini 4).
